modules/installation/pages/uyuni-server-setup.adoc

```diff
@@ -0,0 +1,171 @@
+[[server-setup]]
+= Uyuni Server Setup
+
+This section covers {productname} Server setup, using these procedures:
+
+* Start {productname} setup with {yast}
+* Create the main administration account with the {productname} {webui}
+* Name your base organization and add login credentials
+* Synchronize the {sle} product channel from {scc}
+
+
+
+== Set up {productname} with {yast}
+
+This section will guide you through {productname} setup procedures.
+
+.Procedure: {productname} Setup
+. Log in to the {productname} Server and start {yast}.
+
+. In {yast}, navigate to menu:Network Services[Uyuni Setup] to begin the setup.
+
++
+// To be verified
```

Comment:
Right, needs to be verified before merging.

Reply:
verified